tracer/test/test-applications/security/aspnet/Samples.AspNetMvc5/Samples.AspNetMvc5.csproj

```diff
@@ -192,7 +192,7 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <SecurityLibraryFiles Include="..\..\..\..\..\..\packages\libddwaf.1.0.16\runtimes\**\*.*" />
+    <SecurityLibraryFiles Include="..\..\..\..\..\..\packages\libddwaf.1.0.17\runtimes\**\*.*" />
```

Comment:
I think I removed it in the just merged rate limiter PR cause we don't really need it right?